airflow/providers/databricks/hooks/databricks.py

```diff
@@ -64,10 +66,12 @@
 class RunState:
     """Utility class for the run state concept of Databricks runs."""
 
-    def __init__(self, life_cycle_state: str, result_state: str, state_message: str) -> None:
+    def __init__(
+        self, life_cycle_state: str, state_message: str, result_state: str = '', *args, **kwargs
+    ) -> None:
```

Comment:
Why flipping the arguments? This feels like a source for unnecessary trouble. Also, why take `*args, **kwarg` and ignore them? This is usually an anti-pattern.

Reply:
Please refer to [the comment above](https://github.com/apache/airflow/pull/19835#discussion_r757859224), replied there earlier about the same